src/hooks/useAvaliableThemesFromPoaps.ts

```diff
@@ -14,6 +14,13 @@ const themeToPoapMapping: { theme: NimiCuratedTheme; eventId: number[] }[] = [
   { theme: NimiThemeType.DEVCON, eventId: [60695, 73449] },
   { theme: NimiThemeType.RAAVE, eventId: [63182] },
   { theme: NimiThemeType.DAIVINITY, eventId: [74051] },
+  {
+    theme: NimiThemeType.ETHTLV_2023,
+    eventId: [
+      63182, 98806, 98810, 98812, 98813, 99047, 98815, 98818, 99044, 99608, 99923, 98823, 98826, 99610, 98827, 99348,
```

Comment:
I also added raave eventId just so we can test it out we should remove this before proceeding